src/main/drivers/dshot_bitbang.c

```diff
@@ -501,6 +501,8 @@ static bool bbUpdateStart(void)
         const timeMs_t currentTimeMs = millis();
 #endif
         timeUs_t currentUs = micros();
+        uint32_t totalErpm = 0;
```

Comment:
`uint16_t` is used in other parts of the code to bind `erpm`.

Reply:
I used uint32_t there because to calculate average I have to add erpm values in all motors before dividing by the number of motors, before dividing the value can be over 65535